lib/modules/manager/maven/index.ts

```diff
@@ -6,7 +6,11 @@ export { extractAllPackageFiles } from './extract';
 export { bumpPackageVersion, updateDependency } from './update';
 
 export const defaultConfig = {
-  fileMatch: ['(^|/|\\.)pom\\.xml$', '^(((\\.mvn)|(\\.m2))/)?settings\\.xml$'],
+  fileMatch: [
+    '(^|/|\\.)pom\\.xml$',
+    '^(((\\.mvn)|(\\.m2))/)?settings\\.xml$',
+    '(^|/)(\\.mvn)/extensions\\.xml$',
```

Comment:
```suggestion
    '(^|/)\\.mvn/extensions\\.xml$',
```
don't use captures when not used. (perf)